Before: 49e9d461c87f849db88ea6c22415bfce83309bf85124719359c29cde88f0f050
After: b73de844401e0401e0c97dc556286c7192a90ac56bf4c3a0a7051f44386d8fdb
Patch: c8a647304ae150d4b73d1bee4e81e2615b5016d49a25e1198df88ada3d5a5d3c
--- corvid-features/corvid.rb	2012-08-09 13:21:35.793372509 +1000
+++ corvid-features/corvid.rb	2012-08-09 13:21:53.896804544 +1000
@@ -1,11 +1,4 @@
 install {
   copy_file       'Rakefile'
   empty_directory 'lib'
-  #empty_directory 'wtffff'
-}
-
-update {|ver|
-  if ver == 2
-    empty_directory 'wtffff'
-  end
 }
